Content.Server/SimpleStation14/EndOfRoundStats/Command/EORStatsListCommand.cs

```diff
@@ -0,0 +1,42 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Robust.Shared.Console;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.SimpleStation14.EndOfRoundStats.Command;
+
+[AdminCommand(AdminFlags.Admin)]
+public sealed class EORStatsCommand : IConsoleCommand
+{
+    [Dependency] private readonly IEntityManager _entities = default!;
+    [Dependency] private readonly IEntitySystemManager _entitysys = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+    public string Command => "eorstatslist";
+    public string Description => "Lists the current command-added end of round stats to be displayed.";
+    public string Help => $"Usage: {Command}";
+
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        var _stats = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<CommandStatSystem>();
+
+        if (args.Length != 0)
+        {
+            shell.WriteError("Invalid amount of arguments.");
+            return;
+        }
```

Comment:
unneeded

Reply:
I like the commands being a little more strict. It's unneeded, but makes clear what's happening. Why are you passing additional arguments?

Reply:
why does it need to be strict? arguments aren't used, no reason to complain about their validity

Reply:
It's to clarify usage (for instance, /kill might kill the user of the command only, but passing /kill <other username> should _not_ do that, just to avoid accidental misuse.). I realize that this command is super low risk, but I like following precedence.